src/markitdown/_markitdown.py

```diff
@@ -13,7 +13,8 @@
 import tempfile
 import traceback
 import zipfile
-from typing import Any, Dict, List, Optional, Union
+from typing import Any, Dict, List, Optional, Union, overload
```

Comment:
what is the purpose of  `overload`?

Reply:
i forget to remove it after trying to use different approach